mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,278 @@
+            # Baselines are subtracted before scaling the data.
+            baselines.append(np.fromfile(fid, dtype='i2', count=1)[0])
+            fid.seek(data_offset + 75 * ch_idx + 59)
+            sensitivity = np.fromfile(fid, dtype='f4', count=1)[0]
+            fid.seek(data_offset + 75 * ch_idx + 66)
+            size.append(np.fromfile(fid, dtype='u1', count=5))
+            fid.seek(data_offset + 75 * ch_idx + 71)
+            cal = np.fromfile(fid, dtype='f4', count=1)
+            cals.append(cal * sensitivity * 1e-6 / 204.8)
+
+        fid.seek(event_offset)
+        event_type = np.fromfile(fid, dtype='<i1', count=1)[0]
+        event_size = np.fromfile(fid, dtype='<i4', count=1)
+        if event_type == 1:
+            event_bytes = 8
+        elif event_type == 2:
+            event_bytes = 19
+        else:
+            raise IOError('Unexpected event size.')
+        n_events = event_size // event_bytes
+        events = list()
+        for i in range(n_events):
+            fid.seek(event_offset + 9 + i * event_bytes + 4)
+            offset = np.fromfile(fid, dtype='<i4', count=1)
+            events.append((offset - 900 - 75 * n_channels) //
+                          (n_channels * 2))
+        stim_channel = np.zeros(n_samples)
+
+        for event in events:
+            stim_channel[event - 1] = 1
+
+    info = _empty_info(cnt_info['sfreq'])
+    if lowpass_toggle == 1:
+        info['lowpass'] = highcutoff
+    if highpass_toggle == 1:
+        info['highpass'] = lowcutoff
+    info.update(filename=input_fname,
+                meas_date=calendar.timegm(date.utctimetuple()),
+                description=None, buffer_size_sec=10.)
+
+    coords = _topo_to_sphere(pos)
+    for idx, ch_name in enumerate(ch_names):
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_EEG_CH
+        loc = np.zeros(12)
+        loc[:3] = coords[idx]
+
+        chan_info = {'cal': cals[idx], 'logno': idx + 1, 'scanno': idx + 1,
+                     'range': 1.0, 'unit_mul': 0., 'ch_name': ch_name,
+                     'unit': FIFF.FIFF_UNIT_V,
+                     'coord_frame': FIFF.FIFFV_COORD_HEAD, 'loc': loc,
+                     'coil_type': ch_coil, 'kind': ch_kind}
+        chs.append(chan_info)
+
+    # Add the stim channel.
+    chan_info = {'cal': 1.0, 'logno': len(chs) + 1, 'scanno': len(chs) + 1,
+                 'range': 1.0, 'unit_mul': 0., 'ch_name': 'STI 014',
+                 'unit': FIFF.FIFF_UNIT_NONE,
+                 'coord_frame': FIFF.FIFFV_COORD_UNKNOWN, 'loc': np.zeros(12),
+                 'coil_type': FIFF.FIFFV_COIL_NONE, 'kind': FIFF.FIFFV_STIM_CH}
+    chs.append(chan_info)
+    baselines.append(0)  # For stim channel
+    cnt_info.update(baselines=np.array(baselines), n_samples=n_samples,
+                    n_channels=n_channels, stim_channel=stim_channel)
+    info['chs'] = chs
+    info._check_consistency()
+    return info, cnt_info
+
+
+def _topo_to_sphere(pos):
```

Comment:
@jasmainak I couldn't quite understand how to use ``_topo_to_sphere`` in ``transforms.py``, so I made my own. I think we should have a general function that would just take xy and return xyz like this here. We could just have polymorphic ``_topo_to_sphere`` that would use this if a list is passed (and if this is good enough for you guys) and the other if an angle and a radius is passed. Wdyt?